src/Translator/assets/src/translator_controller.ts

```diff
@@ -7,4 +7,168 @@
  * file that was distributed with this source code.
  */
 
-export * from './translator';
```

Comment:
Conflicted with `options.bundle` from ESBuild, but in reality it didn't bring any value to externalize the translator logic if the final goal is to export everything in the controller. Just define everything in the controller, that's simpler.